server/src/data/agents-channels-data.js

```diff
@@ -0,0 +1,58 @@
+const _ = require('lodash');
+
+const models = require("../models");
+
+class AgentsChannelsData {
+
+  /**
+   * @tested
+   * @param {string} channelId
+   * @param {string} agentId
+   * @returns {Promise<Model>}
+   */
+  async createOne(channelId, agentId) {
+    const input = { channelId, agentId };
+    await this.validate(input);
```

Comment:
I don't like the ``validate`` name. Maybe something like ``agentHasChannelAccess`` or something else that's more semantic?